### ISSUE TITLE
Need to add dropout layers to the model

### ISSUE THREAD

**Issue by alexdauenhauer** [tyu0912/MIDS_W207_facial_recognition#23]

**Comment by alexdauenhauer** [comment]
My understanding was that this would help with overfitting to the training data, but I have yet to find it helping. I propose leaving this open or trying to understand more, but leaving out dropout layers until we haev more time to understand what they are doing
